baseConfig.json

```diff
@@ -291,6 +290,15 @@
         "require-await": "off",
         "sort-keys": "off",
         "symbol-description": "off",
-        "template-tag-spacing": "off"
+        "template-tag-spacing": "off",
+        "for-direction": "off",
+        "no-buffer-constructor": "off",
```

Comment:
I think it would make sense only for nodejs environments. Currently we don’t provide a nodejs specific environment.